apps/patient/src/views/StatusV2.tsx

```diff
@@ -255,10 +223,18 @@ export const StatusV2 = () => {
               </Heading>
               <Card>
                 <VStack w="full" spacing={0}>
-                  {pharmacyInfo}
+                  {displayPharmacy && (
```

Comment:
If `displayPharmacy` is empty (not sure when that would happen, but 🤷) we shouldn't show this whole Pharmacy block (i.e. don't show the `Heading` and the `Card`)

Reply:
ah yeah good catch